### DISCUSSION TITLE
a codemod for easier migration from v13 to v14

### DISCUSSION THREAD

**Discussion by kasra-r77** [testing-library/user-event#1169] [Ideas]
Hi guys,
I recently helped my team migrate to v14 from v13 , and wrote a simple codemod that helps with transforming to new `Promise` based API for the library. Thought sharing it here and give it a bigger transparency for the community. it can also be a part of the docs as a suggestion for the migration to help everyone having a smoother transition.

Here is the link for it: https://github.com/kasra-r77/user-event-update-codemod

I am open to suggestions as well if needed

**Comment by drewdaemon** [comment]
This is awesome. Wish I'd found it two days ago 😅
